### PR TITLE
remove references to personal repositories

### PR DESCRIPTION
**Pull request by bparees**
in partial fulfillment of https://github.com/openshift/origin/issues/9899

### CODE REVIEW

**Comment by bparees** [comment]
[test]
[testextended][extended:core(builds)]

File: test/cmd/newapp.sh [hunk 1]
```diff
@@ -350,8 +350,6 @@ os::cmd::expect_success 'oc delete secret dbsecret'
 os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample -l app=helloworld -o name' 'service/frontend'
 os::cmd::expect_success 'oc delete all -l app=helloworld'
 os::cmd::expect_success 'oc delete secret dbsecret'
-# create from template with code explicitly set is not supported
-os::cmd::expect_failure 'oc new-app ruby-helloworld-sample~git@github.com:mfojtik/sinatra-app-example'
```
Comment (bparees) [inline test/cmd/newapp.sh]: this was a totally broken test so i don't feel much guilt about just removing it outright.

**Comment by bparees** [comment]
@openshift/devex ptal

**Comment by smarterclayton** [comment]
Lgtm

**Comment by bparees** [comment]
[merge]

File: test/cmd/newapp.sh (hunk 1 above)
Comment (stevekuznetsov) [inline test/cmd/newapp.sh]: I've considered removing `os::cmd::expect_failure` for this reason...

**Pull request merged**